### DISCUSSION TITLE
Interpolated classes for things like grid row

### DISCUSSION THREAD

**Discussion by danielr1996** [tailwindlabs/tailwindcss#8101] [Help] [answered]
Somewhere in the documenation I read that you cannot use interpolated class names because the compiler can't recognized those.

So instead of ```<div className={`bg-${success ? 'red' : 'green' }-500`} />``` you must use 
```<div className={success ? 'bg-red-500' : 'bg-green-500' }} />```

But I have the following use case where I need to set the `row-start` property like this

```
{stations.map((station,i) =>
                <li className={`row-start-${i+1}`} key={station.id}>
                    <StationCard {...station}/>
                </li>
            )}
```

But because the values cannot be constructed as strings beforehand this approach doesn't work, are there any solutions for this? 

My current workaround is to justs keep a comment with `row-start-1 row-start-2 .... row-start-100` arround which works because tailwind uses just regex to find which classes are used, but this is rather hacky so I would be happy to hear other approaches

Thanks in advance

**Comment by alexanderbluhm** [comment, marked as answer]
Hi @danielr1996 
A better option than comments in code is the [safelist](https://tailwindcss.com/docs/content-configuration#safelisting-classes) option.
You can also use regular expressions: https://tailwindcss.com/docs/content-configuration#using-regular-expressions

**Reply by danielr1996** [reply]
Thanks for the answer, I must say that I just used an inline style property like `style={{gridRowStart: i+1}}` but it's nice to know that there is a last resort option in tailwind itself
